tasks/lint.yaml

```
@@ -5,9 +5,14 @@ tasks:
   - name: fix
     description: "Fix formatting issues in the repo"
     actions:
+      - description: install codespell deps
+        cmd: CMD=pip && which $CMD || CMD=pip3 && $CMD install codespell
       - description: "Pepr Format"
         cmd: npx pepr format
-
+      - description: Show codespell lint issues
+        cmd: codespell
+      - description: Fix codespell lint issues
+        cmd: codespell -w
```

I kind of like the UX of squishing these two together in a single action (removes the `uds run` boilerplate between them).
```suggestion
      - description: Fix codespell lint issues
        cmd: |
          codespell || true
          codespell -w
```

Okay with either but we will want to add the `|| true` to the `codespell` "show" to ensure that it passes and allows the fix to run.